tests/func/test_cli.py

```diff
@@ -171,3 +173,38 @@ class A:
         args = A()
         with self.assertRaises(DvcException):
             CmdBase(args)
+
+
+class TestUnknownCommandHelp(TestDvc):
+    def test(self):
+        with io.StringIO() as buf, redirect_stdout(buf):
+            try:
+                _ = parse_args(["unknown"])
+            except DvcParserError:
+                pass
+            output = buf.getvalue()
+        with io.StringIO() as buf, redirect_stdout(buf):
+            try:
+                _ = parse_args(["--help"])
+            except SystemExit:
+                pass
+            help_out = buf.getvalue()
+        self.assertEqual(output, help_out)
```

Comment:
Could you please convert these tests to pytest-style? We are moving away from unittest to pytest these days, and all new tests are prefered to be in pytest. Sorry for the confusion :(

Reply:
@efiop, Should I use dvc fixture from tests/dir_helpers instead of TestDvc class? Or something else?

Reply:
@aerubanov Yes, that's the one. But actually, you might not need any `dvc` fixtures at all. Right now I'm not seeing anything that needs dvc-repo. E.g. see `tests/unit/command/test_get.py`.